packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -59,6 +59,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'ssr-guide',
         displayName: 'menu.ssrGuide',
       },
+      {
+        name: 'kitchensink',
+        displayName: 'menu.kitchensink',
+      },
       // GENERATOR_ADD - gettingStarted
```

Comment:
<img width="763" alt="image" src="https://user-images.githubusercontent.com/5394573/189343685-09519742-8728-4d48-beb2-bfac96f9a89a.png">

I don't see a case where you'd need code example for kitchensink, at least we can link docs (but that's also dubiuous)

Reply:
removed